Calculator/index.js

```diff
@@ -1,8 +1,105 @@
 function addEventListenerToButton(element){
     element.addEventListener('click', function(){
-        console.log(element.getAttribute("data-button-value"))
+        let value=element.getAttribute("data-button-value")
+        let type=element.getAttribute("data-button-type")
+
+        switch(type){
+            case "clear": 
+
+                clear();
+                document.getElementById("old-inputs").innerHTML = '';
+                document.getElementById("inputs").innerHTML = '';
+            break;
+
+            case "delete":
+                del();
+            break;
+            
+
+            case "operator":
+
+                numbers[1]=value;
+                if(numbers.length > 3){
+                    operation(numbers);
+                }
+                parenthesis(numbers);
+
+            break;
+
+            case "number":
+
+                numbers[1] ? numbers[2]+=value : numbers[0]+=value;
```

Comment:
even though this logic makes perfect sense, and the line of reasoning is fairly simple, I think one thing that would improve readability _by a lot_ would be to create numerical constants representing the positions in the array, say 
```
const FIRST_NUMBER = 0;
const OPERATOR = 1;
const SECOND_NUMBER = 2;
```
it seems obvious, but if the codebase grows just a little, or you have some of the logic in other files, it can start to get messy really fast